sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSemaphore.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2020,2022, NVIDIA CORPORATION.
```

Comment:
I don't think we normally exclude years when an older file is modified.
```suggestion
 * Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Reply:
I am not a lawyer, but that is a lossy compression, but from what I have seen it appears to be fine.